### ISSUE TITLE
Workspace option doesn't seem to work 

### ISSUE THREAD

**Issue by marinsokol5** [raineorshine/npm-check-updates#1304]
- [X] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [X] I am using the latest version of `npm-check-updates`
- [X] I am using `node >= 14.14`

---

## Description

Hey, thanks for taking time to do this package, it's really useful!! 
I seem to have some problems with using it though in NPM workspaces, I tried reading through the README and through your pitch for the new API in https://github.com/raineorshine/npm-check-updates/issues/896#issuecomment-1249563654 but I can't make it work.

## Steps to Reproduce

```
npm 8.19.2
node v16.17.0
npm-check-updates 16.10.12
```

Steps:

- Use npm workspaces, we can use https://github.com/Quramy/npm-ts-workspaces-example an an example, so let's `git clone git@github.com:Quramy/npm-ts-workspaces-example.git` && `cd npm-ts-workspaces-example`  && `npm install`
- `npx npm-check-updates@16.10.12 -w @quramy/x-cli` (you can also install it globally and run it, both will not work)

## Current Behavior
- command just exits

## Expected Behavior
- command should write which packages from that workspace package could be updated

## Notes
- option `-ws` works fine, like `npm exec npm-check-updates@16.10.12  -- -ws `
- I also noticed that the `--peer` option works incorrectly for the workspaces as it fails to find packages installed in the local node_modules folder of a particular workspace subpackage. Like the way node_modules resolution in workspaces works is:
```
.
├── node_modules/ (storing majority of packages for this NPM workspaces monorepo)
├── package-lock.json
├── package.json
├── packages
│   ├── package-A
│   │   ├── node_modules/ (storing some of the packages for package-A which are specific to it)
│   │   ├── package.json
│   └── package-B
│   │   ├── node_modules/ (storing some of the packages for package-B which are specific to it)
│   |   ├── package.json
```
Currently it just tries to find them in the root node_modules folder but it doesn't check Package-A and Package-B node_modules .

**Comment by raineorshine** [comment]
You should specify the package folder name rather than the full package name:

```
ncu -ws x-cli
```

However, I see no reason why we should not also allow the package name. Sorry for the confusion.

> * I also noticed that the `--peer` option works incorrectly for the workspaces as it fails to find packages installed in the local node_modules folder of a particular workspace subpackage. Like the way node_modules resolution in workspaces works is:

This sounds like a bug. I'm not as familiar with the `--peer` option since it was written by a different contributor, but please open a separate issue with this information and hopefully someone will look into it.

**Comment by marinsokol5** [comment]
>You should specify the package folder name rather than the full package name.

Nice, this works!! 
Yeah it's fine, it's just it might be better to document this as usual workspaces command do work with the name from the package.json.

**Comment by raineorshine** [comment]
Yes, I agree. I think the behavior may even be a regression from #1217. I think it is ignoring entire workspaces instead of ignore local dependencies on workspaces.

**Comment by raineorshine** [comment]
You should be able to do `ncu -w "@quramy/x-cli"` now.

Published in `v16.10.13`
